packages/i18n-utils/src/localize-url.tsx

```diff
@@ -85,6 +85,7 @@ const urlLocalizationMapping: UrlLocalizationMapping = {
 	'wordpress.com/tos/': prefixLocalizedUrlPath( magnificentNonEnLocales ),
 	'wordpress.com/wp-admin/': setLocalizedUrlHost( 'wordpress.com', magnificentNonEnLocales ),
 	'wordpress.com/wp-login.php': setLocalizedUrlHost( 'wordpress.com', magnificentNonEnLocales ),
+	'wordpress.com/refer-a-friend/': setLocalizedUrlHost( 'wordpress.com', magnificentNonEnLocales ),
```

Comment:
Its best if we don't map specific files like this. I think in this case we need to update the referral link we send from the backend to support localization better.